ada_feeding_action_select/config/policies.yaml

```diff
@@ -62,7 +62,7 @@ ada_feeding_action_select:
         - index
       kwargs:
         library: config/acquisition_library.yaml # path to library of actions (package share)
-        index: 0 # Which action to execute
+        index: 1 # Which action to execute
```

Comment:
Do we want to change the default hardcoded action to 1?

Reply:
@amalnanavati Switch back to action 0, but added launch file argument to change it

`action:=<index>` will override the constant policy's action without affecting other policies